storage/binfmt-misc/README.md

```diff
@@ -0,0 +1,17 @@
+# binfmt misc
```

Comment:
I don't think this is `storage` (I mean folder)

Reply:
I am not sure the best place for it. May be we will create the folder like `modules`?

Reply:
I moved it to `misc`